components/ShortEventCard/ShortEventCard.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { StyleSheet, View, Image, Text, ScrollView } from 'react-native';
+
+export default function ShortEventCard() {
+  return (
+    <View>
+      <Image
+        source={require('../../assets/fake_map.png')}
+        style={{width: 325, height: 300, marginBottom: 20, position: 'relative', top: 30}}/>
+      <ScrollView style={styles.scrollView}>
+        <View style={styles.smallCardContainer}>
+          <Text style={styles.titleText}>Volleyball</Text>
+          <Text style={styles.text}>Location: Wash Park</Text>
+          <Text style={styles.text}>Time: 2:00PM</Text>
```

Comment:
I like incorporating the time onto the cards. We could also implement that onto the pins when a user clicks on those on the map